app/controllers/submission_viewing_events_controller.rb

```diff
@@ -0,0 +1,2 @@
+class SubmissionViewingEventsController < ApplicationController
```

Comment:
Auto generated file from 'rails generate model' .  Not used currently, but may be helpful for future work with data table.